lib/Db/Schema.php

```diff
@@ -76,29 +76,30 @@ public function hydrate(array $object): self
 	public function jsonSerialize(): array
 	{
         $properties = [];
-        foreach ($this->properties as $key => $property) {
-            $properties[$key] = $property;
-            if (isset($property['type']) === false) {
-                $properties[$key] = $property;
-                continue;
-            }
-            switch ($property['format']) {
-                case 'string':
-                // For now array as string
-                case 'array':
-                    $properties[$key]['default'] = (string) $property;
-                    break;
-                case 'int':
-                case 'integer':
-                case 'number':
-                    $properties[$key]['default'] = (int) $property;
-                    break;
-                case 'bool':
-                    $properties[$key]['default'] = (bool) $property;
-                    break;
-
-            }
-        }
+		if (isset($this->properties) && $this->properties !== null) {
+			foreach ($this->properties as $key => $property) {
+				$properties[$key] = $property;
+				if (isset($property['type']) === false) {
+					$properties[$key] = $property;
+					continue;
+				}
+				switch ($property['format']) {
+					case 'string':
+					// For now array as string
+					case 'array':
+						$properties[$key]['default'] = (string) $property;
+						break;
+					case 'int':
+					case 'integer':
+					case 'number':
+						$properties[$key]['default'] = (int) $property;
+						break;
+					case 'bool':
+						$properties[$key]['default'] = (bool) $property;
+						break;
+				}
```

Comment:
Switch ends without a default 🤔 it probably should have a default:

Reply:
Bij deze toevaligerwijs niet omdat die alleen iets moet doen als format is gezet. Tis dus okey als die niks doet. Kan aan het einde wel een break zetten maar voelt wat zinloos.